app/src/main/java/lnmiit/com/lnmiit/Splash.java

```diff
@@ -0,0 +1,37 @@
+package lnmiit.com.lnmiit;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+
```

Comment:
refactoring needed! irrelevant spacing.
Also, add your name as author and date.